src/component/todoList.js

```diff
@@ -0,0 +1,68 @@
+export default function TodoList(app) {
+	this.$todoList = document.querySelector("#todo-list");
+	this.todoItems = [];
+
+  this.setState = updatedTodoItems => {
+    this.todoItems = updatedTodoItems;
+    this.render(this.todoItems);
+  };
+
+  this.render = items => {
+    const template = items.map(todoItemTemplate);
+    this.$todoList.innerHTML = template.join("");
```

Comment:
```suggestion
    this.$todoList.innerHTML =  items.map(todoItemTemplate).join("");
```
`map` 은 배열을 반환하니깐 chaining 으로 바로 `join`을 하는건 어떨까요?